config/routes.rb

```diff
@@ -8,6 +8,18 @@
   get '/pl/cieszyn/:path' => redirect('/pl')
   get '/service' => redirect('/')
 
+  namespace :external_link do
+    resources :blog, only: :show
+    resources :facebook, only: :show
+    resources :github, only: :show
+    resources :instagram, only: :show
+    resources :linkedin, only: :show
+    resources :twitter, only: :show
+    resources :vimeo, only: :show
+    resources :website, only: :show
+    resources :youtube, only: :show
+  end
```

Comment:
This requires many controllers. Consider whether it would be easier to have just one controller with one action. The idea is that you can have a list of social links (not really an appropriate name when considering home pages and blogs) and this is part of the route path. For example:
```
torrocus/github
womanonrails/blog
```
But the general template for these routes might be:
```
/:person_slug/:social_slug
```